### DISCUSSION TITLE
Error: Taken Down From Chrome Web Store?

### DISCUSSION THREAD

**Discussion by AnxiousPleaseHelp** [elementdavv/internet_archive_downloader#52] [General]
This is strange.

I'm able to access Internet Archive Downloader from the Edge Extensions Store and Firefox Extensions Store, but it's disappeared from The Chrome Web Store?

This isn't an issue for me but I thought I'd bring it to the attention of the community.

**Comment by elementdavv** [comment]
Unfortunately it is removed from Chrome Web Store.

**Reply by AnxiousPleaseHelp** [reply]
> Unfortunately it is removed from Chrome Web Store.

Thanks!

**By the way did you receive my PayPal donation? I can try again if you didn't receive the payment. I don't know how else to contact you on here. What's the best way to contact you?**

I just checked again and it's still not available... also I saw someone write this in the Firefox extension reviews:
_"Ok for those who just downloaded this in the last week maybe yes this stopped working, Ive been using this and can say 100 percent it worked perfect. Just login, click on borrow a book to open it in reader view and the download button shows under the book. Please fix if possible. Yes it says Blocked now when trying to download."_

**Reply by elementdavv** [reply]
Yes. I have received your donation on July 14, and have also replied to you on July 18 with your email which you left on Paypal.

You can always reach me at [elementdavv@hotmail.com](mailto:elementdavv@hotmail.com). Thank you again.

**Reply by elementdavv** [reply]
> 
As for the review, the Firefox extension works the same all the time.

**Reply by AnxiousPleaseHelp** [reply]
> Yes. I have received your donation on July 14, and have also replied to you on July 18 with your email which you left on Paypal.
> 
> You can always reach me at [elementdavv@hotmail.com](mailto:elementdavv@hotmail.com). Thank you again.

Sorry, my fault!

I see your email message now.

I'll donate as much as I can.

**Reply by AnxiousPleaseHelp** [reply]
Is archive.org down for everyone else? It's been offline for a couple days at least now.

Does archive.org being offline have anything to do with the internet archive downloader extension? 😭 

> > 
> 
> As for the review, the Firefox extension works the same all the time.

Is archive.org down for everyone else? It's been offline for a couple days at least now.

Does archive.org have anything to do with the internet archive downloader extension? 😭

**Reply by elementdavv** [reply]
> Is archive.org down for everyone else? It's been offline for a couple days at least now.

Just now It has half revived read only, book pages can not be opened. And I noticed that the availability filter has been removed, there may be some strategy changes in the future.

> Does archive.org have anything to do with the internet archive downloader extension? 😭

No. It was attacked by ddos. Their all services were down, wayback mechine, blog etc, not just archive.org.

**Reply by AnxiousPleaseHelp** [reply]
> > Is archive.org down for everyone else? It's been offline for a couple days at least now.
> 
> Just now It has half revived read only, book pages can not be opened. And I noticed that the availability filter has been removed, there may be some strategy changes in the future.
> 
> > Does archive.org have anything to do with the internet archive downloader extension? 😭
> 
> No. It was attacked by ddos. Their all services were down, wayback mechine, blog etc, not just archive.org.

Thanks!

I'm trying to donate to thearchive.org but I can't even login...

**Reply by elementdavv** [reply]
> I'm trying to donate to thearchive.org but I can't even login...

It seems revived today. Everything works properly except the missing availability search filter.

**Reply by AnxiousPleaseHelp** [reply]
> > I'm trying to donate to thearchive.org but I can't even login...
> 
> It seems revived today. Everything works properly except the missing availability search filter.

Okay I did a search and came across this fraudster: https://chromewebstore.google.com/detail/internet-archive-download/gggkkannodcpddhlkladmnmfbbeooogj

He's using the same name of your extension and even the same logo...  I reported him to Google

For other users on here: be careful of this guy and other fakesters!

**Reply by AnxiousPleaseHelp** [reply]
> > I'm trying to donate to thearchive.org but I can't even login...
> 
> It seems revived today. Everything works properly except the missing availability search filter.

I just checked this again and it’s been deleted. This is what is displayed now:

**_“This item is not available”_**

**Reply by AnxiousPleaseHelp** [reply]
Is there any chance that this extension will be allowed back onto the official Chrome Web Store?

I've reported the fake/scam versions of Internet Archive Downloader to Google for you

**Reply by elementdavv** [reply]
> Is there any chance that this extension will be allowed back onto the official Chrome Web Store?
> 
> I've reported the fake/scam versions of Internet Archive Downloader to Google for you

Thank you!
I have reported several weeks ago. but they refuse to change.
